### PR TITLE
Hdb 3 patient details

### PR DESCRIPTION
**Pull request by assawalhy**

### CODE REVIEW

File: app/[locale]/globals.css
Comment (mohamed-ahmed-yousef) [inline app/[locale]/globals.css:1]: Is we Used this file?

File: app/[locale]/dashboard/patient/[patientId]/page.tsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+	params: { patientId },
+}: { params: { patientId: string } }) {
+	const avatar = useAvatar(undefined, { seed: patientId, size: 100 });
+	return (
+		<>
+			<Breadcrumbs>
+				<Anchor href={"/dashboard"}>Dashboard</Anchor>
+				<Anchor href={"/dashboard/patients"}>Patients</Anchor>
+				<span>Patients {patientId}</span>
+			</Breadcrumbs>
```
Comment (mohamed-ahmed-yousef) [inline app/[locale]/dashboard/patient/[patientId]/page.tsx:37]: why didn't you use the next-intl `Link` tag?

File: tailwind.config.ts
Comment (mohamed-ahmed-yousef) [inline tailwind.config.ts:1]: I think we didn't use the shadcn UI configuration, so we can remove it

**Review by mohamed-ahmed-yousef — COMMENTED** [review]
At all LGTM.

File: app/[locale]/dashboard/patient/[patientId]/page.tsx (hunk 1 above)
Comment (assawalhy) [reply app/[locale]/dashboard/patient/[patientId]/page.tsx:37]: Anchor will work here like `Link` but will provide the design and UI of the link which adhere to Mantine design system.

File: tailwind.config.ts
Comment (assawalhy) [reply tailwind.config.ts:1]: Exactly, will remove it!

File: app/[locale]/globals.css
Comment (assawalhy) [reply app/[locale]/globals.css:1]: Tailwind is used in combination with Mantine. I believe we can configure Tailwind to match the Mantine styles like shadow and colors but we have no time to spend fine-tuning here.

**Comment by assawalhy** [comment]
Please merge if all look good to you.  @mohamedahmed-cloud

**Review by mohamed-ahmed-yousef — APPROVED** [review]

**Pull request merged**